Meta.Tests.ps1

```diff
@@ -60,6 +28,9 @@ if (($env:PSModulePath.Split(';') | Select-Object -First 1) -ne $pwd)
     $env:PSModulePath = "$pwd;$env:PSModulePath"
 }
 
+# Make sure MetaFixers.psm1 is loaded - it contains Get-TextFilesList
+Import-Module -Name (Join-Path -Path $PSScriptRoot -ChildPath 'MetaFixers.psm1') -Force
```

Comment:
It's good practice to leave all Import-Module at the top of the script to easily find out what dependencies script uses. Can you please move it back?